telemetry/telemetry.go

```diff
@@ -383,3 +385,21 @@ func (reportMgr *ReportManager) ReportToBytes() ([]byte, error) {
 	report, err = json.Marshal(reportMgr.Report)
 	return report, err
 }
+
+func SendCNIMetric(cniReport *CNIReport, tb *TelemetryBuffer) error {
```

Comment:
instead of creating different functions for CNS, such as SendCNSMetric, can this be just one function like SendMetric and takes the type / *Report.
Can it take *Report to reuse the func signature between CNI, CNS ?

Reply:
nope. CNI uses telemetry service but not cns